README.md

```diff
@@ -18,10 +18,198 @@ Or install it yourself as:
+
+On the HAProxy server, add `agent-check agent-port 8080 agent-inter <seconds>s` to the config line for each server listed for that backend. This tells HAProxy to query port 8080 on the backend every `<seconds>` seconds for health information.
+
+### REST API
+
+- /
+  * GET: Returns table with status of each service.
+
+		Litmus Paper 1.1.1
+
+		 Service    │ Reported │ Measured │ Health
+		 Name       │ Health   │ Health   │ Forced?
+		────────────┴──────────┴──────────┴─────────
+		 myapp            0        100      Yes, Reason: testing
+		 myotherapp      72         72      No
+
+- /down
+  * POST: Creates a global force down. Parameters: reason => reason for the force down.
+  * DELETE: Deletes global force down, if one is in place. Any service-specific force downs remain in effect.
+
+- /up
+  * POST: Creates a global force up. Parameters: reason => reason for the force up.
+  * DELETE: Deletes global force up, if one is in place. Any service-specific force ups remain in effect.
+
+- /health
+  * POST: Creates a global force health. Parameters: reason => reason for the force health, health => health to force to.
+  * DELETE: Deletes global force health, if one is in place. Any service-specific force healths remain in effect.
+
+- /`<service>`/status
+  * GET: Returns a detailed status output for `<service>`, including all the subchecks. This output easier to parse than the output from GET /. Also sets X-Health and X-Health-Forced headers in the response.
+
+    ```
+    Health: 82
+    Measured Health: 82
+    Dependency::HTTP(http://localhost/heartbeat): OK
+    Metric::CPULoad(100): 82
+    ```
+
+- /`<service>`/down
+  * POST: Creates a force down just for `<service>`. Parameters: reason => reason for the force down.
+  * DELETE: Deletes force down for `<service>`, if one is in place. Global force downs remain in effect.
+
+- /`<service>`/up
+  * POST: Creates a force up just for `<service>`. Parameters: reason => reason for the force up.
+  * DELETE: Deletes force up for `<service>`, if one is in place. Global force ups remain in effect.
+
+- /`<service>`/health
+  * POST: Creates a force health just for `<service>`. Parameters: reason => reason for the force health, health => health to force to.
+  * DELETE: Deletes force health for `<service>`, if one is in place. Global force healths remain in effect.
+
+### Litmusctl
+
+There is also a CLI included called `litmusctl`.
+
+This has the same functionality as the rest API:
+- `litmusctl list` = `GET /`
+- `litmusctl status <service>` = `GET /<service>/status`
+- `litmusctl force (down|up|health) [health] [-r REASON]` = `POST /(down|up|health)`
+- `litmusctl force (down|up|health) [health] <service> [-r REASON]` = `POST /<service>/(down|up|health)`
+- `litmusctl force (down|up|health) -d` = `DELETE /(down|up|health)`
+- `litmusctl force (down|up|health) <service> -d` = `DELETE /<service>/(down|up|health)`
+
+## Tests
+
+Run tests using `rake`. The default task runs all the tests.
 
```

Comment:
Maybe we could add a TODO section? It might invite contributions. Some thoughts for what could go in there...
```markdown
TODO:
1. Accept configuration in either YAML or ruby format.
2. Improve concurrency model, with a health-check process and a responder process.
3. Improve concurrency of agent-check daemon.
```

Reply:
Another cool option, if we have more time later - add a vagrant file or something that configures the daemon and a few examples, so that people can play around with it!

Reply:
I started spiking out a vagrant file for doing this and for doing benchmarking across branches when we were looking at adding caching - I'll see if I can dig that work up and revive it.

Reply:
I added this in https://github.com/braintree/litmus_paper/pull/34 - maybe we can just merge and iterate on that (I also found it really useful for testing and running the specs, which are not happy on OS X).